src/constants/networks.ts

```diff
@@ -112,10 +108,10 @@ export const FALLBACK_URLS = {
   [ChainId.BASE]: [
     // "Safe" URLs
     'https://mainnet.base.org',
-    // "Unsafe" URLs
-    QUICKNODE_BASE_RPC_URL,
-    'https://base-mainnet.blastapi.io/b5a802d8-151d-4443-90a7-699108dc4e01',
-    'https://svc.blockdaemon.com/base/mainnet/native?apiKey=zpka_1334e7c450464d06b6e33a972a7a4e57_75320f43',
+    'https://base.gateway.tenderly.co',
+    'https://base.publicnode.com',
+    'https://1rpc.io/base',
+    'https://base.meowrpc.com',
```

Comment:
<img width="455" alt="image" src="https://github.com/Uniswap/interface/assets/5773490/e5e7e2a6-cd6b-4099-bd51-d1a01ce07e57">

do we want to include the 1rpc and meowrpc urls? I don't see them in the linked code

https://chainid.network/chains.json